tools/cooja/apps/collect-view/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "java"
+
+libsDirName = ".."
+
+repositories {
+    jcenter()
+}
```

Comment:
I have no idea whether I'm right, but aren't repositories also imported via the dependencies? And is it needed at all?